### ISSUE TITLE
Squiz.WhiteSpace.FunctionSpacing.After on last method in class/interface

### ISSUE THREAD

**Issue by o5** [squizlabs/PHP_CodeSniffer#1889]
I would like to have only one blank line as separation in class/interface between methods. 

I found a sniff `Squiz.WhiteSpace.FunctionSpacing` which provides a settings for number of `spaces `. It works fine except requiring blank line after last method in class.

This snippet does not pass:

```php
class Foo
{
       function bar()
       {
       }

       function foo()
       {
       }
}
```

```
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------------------------------------------
 9 | ERROR | [x] Expected 1 blank line after function; 0 found (Squiz.WhiteSpace.FunctionSpacing.After)
```

I'm also using  `PSR2.Classes.ClassDeclaration.CloseBraceAfterBody` and when I add the line, FunctionSpacing will works but CloseBraceAfterBody does not.

Any workaround? Is possible to add this behaviour? How? :)

Thank you
Peter

**Comment by o5** [comment]
https://github.com/squizlabs/PHP_CodeSniffer/issues/652

Closing, sorry.
